src/components/ui/sliders.js

```diff
@@ -299,3 +299,137 @@ export const TestimonialCard = ({
+      readMoreLink: '#',
+    },
+    {
+      imgSrc: 'https://via.placeholder.com/500x300/00AEEF/ffffff',
+      imgAlt: '',
+      imgW: 500,
+      imgH: 300,
+      title: 'Slide Four',
+      desc: 'Wafer I love icing cookie toffee shortbread I love cookie. Tart liquorice icing croissant topping tootsie roll. Croissant liquorice danish carrot cake I love candy canes tootsie roll. Sugar plum dessert danish caramels icing jelly beans cheesecake jelly. Bonbon danish toffee soufflé danish caramels pie soufflé.',
+      readMoreLink: '#',
+    },
+    {
+      imgSrc: 'https://via.placeholder.com/500x300/EBF3F9/000000',
+      imgAlt: '',
+      imgW: 500,
+      imgH: 300,
+      title: 'Slide Five',
+      desc: 'Toffee dessert ice cream danish pastry sweet roll. Cotton candy I love candy cookie cake. Pastry I love lemon drops cupcake pie oat cake. Fruitcake tiramisu chocolate fruitcake caramels soufflé cake. I love I love bonbon chocolate bar chocolate chocolate cake dragée chupa chups pudding.',
+      readMoreLink: '#',
+    },
+  ];
+  return (
+    <>
+      <ClickableImages
+        controlledSwiper={controlledSwiper}
+        data={data}
+      />
+      <Swiper
+        onSwiper={setControlledSwiper}
+        slidesPerView={1}
+        spaceBetween={30}
+        navigation
+        loop
+        modules={[Navigation]}
+        className='mySwiper'>
+        {data.map((slide, sliderIndex) => (
+          <SwiperSlide key={sliderIndex}>
+            <div className='grid gap-8 px-10 lg:grid-cols-4 xl:gap-16 md:px-16 xl:px-28'>
+              <div className='my-auto col-span-2'>
+                <Image
+                  src={slide.imgSrc}
+                  alt={slide.imgAlt}
+                  width={slide.imgW}
+                  height={slide.imgH}
+                  sizes={'(max-width: 768px) 100vw, (max-width: 1280px) 75vw'}
+                  className=' mx-auto'
+                />
+              </div>
+              <div className='col-span-2 text-center lg:text-left'>
+                <H2 className='mb-8 text-gray-900'>{slide.title}</H2>
+                <BodyText className='text-gray-900'>{slide.desc}</BodyText>
+                {slide.readMoreLink ? (
+                  <Link
+                    href={slide.readMoreLink}
+                    className='block mt-8'>
+                    Read more
+                  </Link>
+                ) : null}
+              </div>{' '}
+            </div>
+          </SwiperSlide>
+        ))}{' '}
+      </Swiper>
+    </>
+  );
+};
+
+/**
+ * @description - this component is a child component of the ControlledSliderWImages component.
+ * @param {state} controlledSwiper - the state variable that is controls the index of the slide
+ * @param {array} data - the information stored in data const
```

Comment:
I think it would be beneficial to describe the expected structure of each element and include an example.